tests/Maker/MakeResetPasswordTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Tests\Maker;
+
+use Symfony\Bundle\MakerBundle\Maker\MakeResetPassword;
+use Symfony\Bundle\MakerBundle\Test\MakerTestCase;
+use Symfony\Bundle\MakerBundle\Test\MakerTestDetails;
+
+class MakeResetPasswordTest extends MakerTestCase
+{
+    public function getTestDetails()
+    {
+        yield 'reset_password' => [MakerTestDetails::createTest(
+            $this->getMakerInstance(MakeResetPassword::class),
+            [
+                'jr@rushlow.dev',
+                'SymfonyCasts',
+                'App\Entity\User',
+                'app_home',
+            ])
+            ->addExtraDependencies('security-bundle')
+            ->setFixtureFilesPath(__DIR__.'/../fixtures/MakeResetPassword'),
```

Comment:
We should be a functional test inside the skeleton. For simplicity, it could *just* go to `/reset-password` and check for a 200 status code. The gold standard would be to actually fill out the form, capture the reset password link somehow, then go to THAT URL and reset the password :). We could do that "gold standard" never or later ;)